packages/loader/container-loader/src/connectionManager.ts

```diff
@@ -743,8 +740,8 @@ export class ConnectionManager implements IConnectionManager {
             return;
         }
 
-        const delayMs = error !== undefined ? getRetryDelayFromError(error) : undefined;
-        if (delayMs !== undefined) {
+        const delayMs = getRetryDelayFromError(error);
+        if (error !== undefined && delayMs !== undefined) {
```

Comment:
is `error !== undefined` redundant here?

Reply:
Correct.  But without it this way, TypeScript can't figure out the type narrowing and complains that `error` could be `undefined` on the next line.  If you have a stronger opinion, I can do `error!` on the next line with a lint error suppression, but I prefer to avoid those myself.